test/smart-wallet/refund-estimation.ts

```diff
@@ -313,6 +319,19 @@ describe("Wallet tx gas estimations with and without refunds", function () {
 
     console.log(refundInfo);
 
+    const dataEstimate = SmartAccount.interface.encodeFunctionData(
+      "execTransaction",
+      [
+        transaction,
+        0, // batchId
+        refundInfo,
+        signature,
+      ]
+    );
+
+    const baseGasReal = txBaseCost(dataEstimate);
```

Comment:
will it be used somewhere?

Reply:
I don't think so. we're trying to get initial gas usage that have to be added in baseGas field of fee refund
initial gas is 21000 + depends on 1 and 0 bytes in msg.data

see now there is a circular dependency cause if you want to use right msg.data you need a signature first, and for this signature you'd need accurate baseGas. but the whole point of estimation is to get right baseGas so you can sign it.